src/activities/act-7-1-2.xml

```diff
@@ -113,18 +113,119 @@
       <ol label="a.">
         <li>
           <p>
-            Solutions for each of the prompts above.
+            For the skydiver:
+            <md>
+              <mrow>\left. \frac{dv}{dt}\right|_{(v = 0.5)} \amp \approx 1.5  \amp   \left. \frac{dv}{dt}\right|_{(v = 1)} \amp \approx 1.2 \amp   \left. \frac{dv}{dt}\right|_{(v = 1.5 )} \amp \approx 0.9</mrow>
+              <mrow>\left. \frac{dv}{dt}\right|_{(v = 2)} \amp \approx 0.6  \amp   \left. \frac{dv}{dt}\right|_{(v = 2.5)} \amp \approx 0.3</mrow>
+            </md>
+          </p>
+        </li>
+
+        <li>
+          <p>
+            For the meteorite:
+            <md>
+              <mrow>\left. \frac{dv}{dt}\right|_{(v = 3.5)} \amp \approx -0.3  \amp   \left. \frac{dv}{dt}\right|_{(v = 4)} \amp \approx -0.6</mrow>
+              <mrow>\left. \frac{dv}{dt}\right|_{(v = 4.5)} \amp \approx -0.9   \amp   \left. \frac{dv}{dt}\right|_{(v = 5)} \amp \approx -1.2</mrow>
+            </md>
+            A graph of the points from parts (a) and (b) is shown in the following diagram:
+              <figure xml:id="F-act-7-1-2-a">
```

Comment:
`xml:id="F-act-7-1-2-a"` is already defined. As a good rule of thumb, don't put things like figures and tables with numbers inside `answer` or `solution`, since then the numbering of the book overall will jump when skipping over such things in the output. Although it won't allow a caption, the best thing to do here is to replace the `figure` with `sidebyside` (and omit `caption`). The caption generally isn't going to be necessary in a solution/answer, so you can get away with `sidebyside` without `caption` and no `@xml:id`.